docs/datacontract.md

```diff
@@ -421,3 +439,17 @@ The top-level key within a FileDefinition serves as the FileDefinition name. Thi
 </table>
 
 
+## Alternative Datacontract locations
+CsvToFHIR uses the [smart_open](https://github.com/RaRe-Technologies/smart_open) library to read the Datacontract and any referenced file definitions within it.
```

Comment:
is this now used for all reading of the data contract? even when not in external cloud storage...is that what we want?

Reply:
It looks like the library provides an "open" function which effectively replaces the "open" function from the base library. We can update our setup here to make cloud storage an "opt-in" feature and then check for it's existence at runtime. 

I know that this complicates things a bit, but in the event that we have issues with the library we will still be able to function.

Reply:
Once the other issues, if any, are addressed, I will be happy to submit some changes for this PR to support the "optional" open.